lldb/test/API/lang/swift/cxx_interop/forward/swift-class-with-cxx-ivars/TestSwiftClassWithCxxIvars.py

```diff
@@ -0,0 +1,35 @@
+
+"""
+Test that a Swift class with C++ ivars is printed correctly
+"""
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+
+
+class TestSwiftClassWithCxxIvars(TestBase):
+
+    @swiftTest
+    def test(self):
+        self.build()
+        self.runCmd('setting set target.experimental.swift-enable-cxx-interop true')
+        _, _, _, _= lldbutil.run_to_source_breakpoint(
+            self, 'Set breakpoint here', lldb.SBFileSpec('main.swift'))
+
+        self.expect('v swiftClass', substrs=['SwiftClass', 'cxxClass', 'a1', '10', 'a2', '20', 'a3', '30',
+            'cxxSubclass', 'a1', '10', 'a2', '20', 'a3', '30', 'a4', '40'])
+        self.expect('p swiftClass', substrs=['SwiftClass', 'cxxClass', 'a1', '10', 'a2', '20', 'a3', '30',
```

Comment:
With dwim-print `p` is effectively becoming an alias for `v` in simple expressions. You want to use `expr` here.

Reply:
@kastiglione